scripts/codegen/CAN/App_CanMsgs.dbc

```diff
@@ -315,6 +315,7 @@ SG_ STACK_WATERMARK_ABOVE_THRESHOLD_TASK1KHZ : 1|1@1+ (1,0) [0|1] "" DEBUG
 SG_ STACK_WATERMARK_ABOVE_THRESHOLD_TASKCANRX : 2|1@1+ (1,0) [0|1] "" DEBUG
 SG_ STACK_WATERMARK_ABOVE_THRESHOLD_TASKCANTX : 3|1@1+ (1,0) [0|1] "" DEBUG
 SG_ WATCHDOG_TIMEOUT : 4|1@1+ (1,0) [0|1] "" DEBUG
+SG_ CHARGING_DONE : 5|1@1+ (1,0) [0|1] "" DEBUG
```

Comment:
shouldnt we see an indication that this is complete when we are no longer in the charge state? I dont think this is needed

Reply:
We should see something like this if the charger is connected: init->precharge->charge->init

So if we monitor the state variable on CAN we will have this information while debugging

Reply:
To connect the comments and suggestions below, I think it's best that we stay in the charge state once charging is complete when the pack voltage is ~400 V. Then, set the CAN signal "CHARGING_COMPLETE". The user will then disconnect the charger which will put the BMS from charge->init state. "CHARGING_COMPLETE" should be cleared as well

Reply:
I agree with tim here. We don't want to re-enter init state then go through precharge again and back to charge state in a loop.

Reply:
hm yeah i can see how the charging complete would be nice to inform the user that the charger needs to be removed

Reply:
@Eddy13013 are there any intuitive ways to inform the user that we have finished charging?

Reply:
@timbitss why is charging done regarded as an AIR shdn fault?

Reply:
I'll just make a separate can msg for it